src/core/lv_obj_pos.h

```diff
@@ -355,7 +355,7 @@ void lv_obj_move_children_by(struct _lv_obj_t * obj, lv_coord_t x_diff, lv_coord
  * @param recursive     consider the transformation properties of the parents too
  * @param inv           do the inverse of the transformation (-angle and 1/zoom)
  */
-void lv_obj_transform_point(const struct _lv_obj_t * obj, lv_point_t * p, bool recursive, bool inv);
+void lv_obj_transform_point(const struct _lv_obj_t * obj, lv_point_t * p, lv_point_t * p1, bool recursive, bool inv);
```

Comment:
Please add a detailed description about the new parameter in the comments above.

Reply:
@param p1             a point to the last untransformed coordinate
Is It okay to describe like this, I am not good at english, please point out any grammatical errors.

Reply:
I'm also not a native speaker but let's try this:
```
p1 a previous untransformed point.  If the object was moving and it was transformed the a previous point helps to fix incorrect calculations due to the transformed coordinate system. 
```

Is it technically correct?

Reply:
>  p a current untransformed point, so (p - p1) can calculate the current moving distance, and pivot + (p - p1) ensures taht the distance between the current pressing point and the starting point of the object is consistent with the previous one.

Please add it as a comment in the function :slightly_smiling_face:

Reply:
I wonder how it works with nested transformation. E.g. is there is a 45 deg rotated rectangle in which there is a -70 deg rotated other rectangle. As transformation is recursive `p` will be transformed during the process.

Reply:
Only focus on the most foreground object when pressing rhe screen. As long as the value of `proc->types.pointer.vect` is correct, no matter how nested transformation, it will work normally when moving the object. What do you think?

But I wonder if `lv_obj_transform_point` is not called in function `indev_proc_press`, will it cause any problems? Or what are your considerations for adding `lv_obj_transform_point` to function `indev_proc_press` at taht time? 

It seems that the value of `proc->types.pointer.vect` can be calculated correctly without calling `lv_obj_transform_point`.